css/style.css

```
@@ -331,8 +331,11 @@ img {
 	margin-top: 0;
 }
 #footer {
-	color: #ccc;
-	font-size: 12px;
+	color: #757575;
```

```suggestion
	color: #5d768e;
```

In terms of HSL, this has same Lightness than #757575 (so essentially same contrast ratio on white), but same Hue and Saturation as the main blue. This ends up feeling less off than pure gray.